### PR TITLE
Feature/81 guzzle exception handling

### PR DESCRIPTION
**Pull request by sjongejan**

### CODE REVIEW

File: app/Request.php [hunk 1]
```diff
@@ -25,6 +25,14 @@ class Request extends Model
         'PATCH',
     ];
 
+    protected $visible = [
```
Comment (FaaPz) [inline app/Request.php]: What does `$visible`?

File: app/Jobs/ExecuteRequestJob.php [hunk 2]
```diff
@@ -42,13 +43,19 @@ public function handle(Client $client)
...
+            $response = $client->request($endpoint->method, $endpoint->url, [
+                'http_errors' => false,
+                'allow_redirects' => [
+                    'max' => 10,
+                    'on_redirect' => $onRedirect,
+                ],
+            ]);
+            $this->handleResponse($response);
+        } catch (RequestException $e) {
+            $this->request->error_message = $e->getMessage();
```
Comment (FaaPz) [inline app/Jobs/ExecuteRequestJob.php:56]: `str_limit` this. We only have 255 characters to save.

**Review by FaaPz — CHANGES_REQUESTED** [review]
Two minor feedback points.

Great findings on the typos and such! 👍

File: app/Jobs/ExecuteRequestJob.php (hunk 2 above)
Comment (sjongejan) [reply app/Jobs/ExecuteRequestJob.php:56]: Lumen does this out-of-the-box.

**Review by FaaPz — APPROVED** [review]

**Pull request merged**